runner/runners/facade.go

```diff
@@ -0,0 +1,13 @@
+package runners
```

Comment:
Can we put a brief comment header on all of the runner/runners/*.go implementation and other files